go/vt/sqlparser/sql.y

```diff
@@ -7142,6 +7142,14 @@ set_expression_assignment:
   {
     $$ = &SetVarExpr{Name: $1, Expr: NewStrVal($3), Scope: SetScope_None}
   }
+| column_name '=' STRING STRING
```

Comment:
This is fine for unblocking you, but it seems like what you actually want is a new rule string_literal or similar, something that reduces any `'a' 'b' 'c'` literal to `'abc'`

E.g this works in mysql but won't here:

```
mysql> select '1' '1' '1'
    -> ;
+-----+
| 1   |
+-----+
| 111 |
+-----+
```

Reply:
Could just TODO it for now and come back later if you want, there are a lot of places STRING is used

Reply:
Thank u! yeah... I briefly thought about the complete change for supporting implicit adjacent quoted string concatenation (https://github.com/dolthub/dolt/issues/5232), but wanted to keep this scoped to the binlog data import problem since I wasn't sure the larger change to support implicit concatenation was worth the extra time right now. I did drop a link to the larger implicit concatenation issue in here though and I'm happy to circle back around to it.